README.md

```diff
@@ -22,35 +22,31 @@ The LSS-ROS2-Arms repository contains common packages that are used by both the
 sudo pip install colcon-common-extensions
 sudo pip install vcstool
 sudo apt install ros-galactic-backward-ros
-sudo apt install python3-rosdep2
```

Comment:
this was not needed, I actually got package conflicts because of it and skipped it. Still all was working.